test/integration/eslint/test/index.test.js

```diff
@@ -122,10 +121,22 @@ describe('ESLint', () => {
       )
       expect(output).toContain('Compiled successfully')
     })
+
+    test('missing Next.js plugin', async () => {
+      const { stdout, stderr } = await nextBuild(dirNoEslintPlugin, [], {
+        stdout: true,
+        stderr: true,
+      })
+
+      const output = stdout + stderr
+      expect(output).toContain(
+        'The Next.js plugin was not detected in your ESLint configuration'
+      )
+    })
   })
 
   describe('Next Lint', () => {
-    test('first time setup', async () => {
+    test('show a prompt to set up ESLint if no configuration detected', async () => {
```

Comment:
Can we update this test to go through the full process, an example of testing this can be seen [here](https://github.com/vercel/next.js/blob/3a9fbd83969f2435b78387ca3f716a3c8dddcb7e/test/integration/create-next-app/index.test.js#L193) when we used to test the prompting for selecting an example in `create-next-app`

Reply:
Thanks for the example :)

For some reason, I couldn't get the action of selecting the prompt to work within a child process in the test and I'm not sure why (arrow key actions worked fine 🤔).  I separately created a `--strict` flag to simplify running `next lint` for the first time for users who want the strict ESLint configuration and don't need to see the prompt, so I opted to use that to simulate ["almost" the full process](https://github.com/vercel/next.js/pull/26584/files#diff-b5278a383ecba4c70b58cc634e50f110b9167cacd2c823b990a5447dc8b103e2R200-R226) and test that the `.eslintrc` is created and `eslint` + `eslint-config-next` are installed successfully. 

I hope this sufices!